### PR TITLE
[11.0][mig ][partner_second_lastname]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration to 11.0

Depends on 

- [x] partner_firstname - #87

### CODE REVIEW

**Review by HviorForgeFlow — APPROVED** [review]
:+1: Functional review

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Comment by pedrobaeza** [comment]
This module is WIP until the dependency is merged.

**Comment by pedrobaeza** [comment]
Sorry, I see that is merged. Then a rebase is needed.

File: partner_second_lastname/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (rgarnau) [inline partner_second_lastname/README.rst:2]: https

**Review by rgarnau — APPROVED** [review]
(just one comment)

**Review by njeudy — COMMENTED** [review]
- Just install and test in a fresh 11.0 database

Ok for me

**Review by njeudy — APPROVED** [review]
- Just install and test in a fresh 11.0 database

Ok for me

**Pull request merged**